File: openbis_api/source/java/ch/ethz/sis/openbis/generic/shared/api/v3/exceptions/NotFetchedException.java
package ch.ethz.sis.openbis.generic.shared.api.v3.exceptions;

import ch.systemsx.cisd.common.exceptions.UserFailureException;

public class NotFetchedException extends UserFailureException
{
    private static final long serialVersionUID = 1L;

    public NotFetchedException(String message, Throwable cause)
    {
        super(message, cause);
    }

    public NotFetchedException(String message)
    {
        super(message);
    }
    
}

File: js-test/servers/common/core-plugins/tests/1/as/webapps/openbis-v3-api-test/html/exceptions/NotFetchedException.js
var NotFetchedException = function(message, cause) {
	this['@type'] = 'NotFetchedException';
    UserFailureException.call(this, message, cause);
};

stjs.extend(NotFetchedException, UserFailureException, [], function(constructor, prototype) {
    constructor.serialVersionUID = 1;
}, {});
